bullet-train.zsh-theme

```diff
@@ -568,11 +603,15 @@ build_prompt() {
   prompt_nvm
   prompt_go
   prompt_git
+  prompt_svn
   prompt_hg
   prompt_cmd_exec_time
   prompt_end
 }
 
+  prompt_end
+}
+
```

Comment:
This shouldn't be here. I ques you didn't checked this file. This needs to be fixed before merging.